src/XMakeBuildEngine/UnitTests/FileLogger_Tests.cs

```diff
@@ -375,7 +381,7 @@ private void VerifyFileContent(string file, string expectedContent)
             }
         }
 
-        #region DistributedLogger
+#region DistributedLogger
```

Comment:
I narrowed it down to either resharper refactorings or intellisense fixes. One of them rewrites the AST and thinks that those `#regions` should be indented.